cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataProcConf.java

```diff
@@ -195,11 +221,15 @@ private static DataProcConf create(Map<String, String> properties, @Nullable SSH
     int masterDiskGB = getInt(properties, "masterDiskGB", 500);
     int workerDiskGB = getInt(properties, "workerDiskGB", 500);
 
-
+    long pollCreateDelay = getLong(properties, "pollCreateDelay", 60);
```

Comment:
Right now - we're putting the default values in two places (here and in the UI json). This might make future maintenance/modifications yield inconsistent defaults.

Reply:
agreed. Not sure the best way to handle this though. We have the same issue with all our plugins.